idx/memory/memory_find_test.go

```diff
@@ -223,3 +290,92 @@ func BenchmarkConcurrent8Find(b *testing.B) {
 	}
 	close(ch)
 }
+
+func ixFindByTag(org, q int) {
+	series, err := ix.IdsByTagExpressions(org, tagQueries[q].Expressions, 0)
+	if err != nil {
+		panic(err)
+	}
+	if len(series) != tagQueries[q].ExpectedResults {
+		for s := range series {
+			memoryIdx := ix.(*MemoryIdx)
+			fmt.Println(memoryIdx.DefById[s.ToString()].Tags)
+		}
+		panic(fmt.Sprintf("%+v expected %d got %d results instead", tagQueries[q].Expressions, tagQueries[q].ExpectedResults, len(series)))
```

Comment:
don't print to stdout and call panic.
use the standard functions https://golang.org/pkg/testing/#TB

Reply:
fixed https://github.com/grafana/metrictank/pull/729/commits/186deabb44a6eb6966bfa7e1b95c147b3fc5f22a